Description: Don't build non-DFSG (GFDL 1.1 licensed) help files
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = data po src man help
+SUBDIRS = data po src man
 
 EXTRA_DIST =			\
 	intltool-extract.in	\
--- a/configure.ac
+++ b/configure.ac
@@ -202,7 +202,6 @@
 data/mate-user-share-webdav.desktop.in
 data/mate-user-share-obexftp.desktop.in
 data/mate-user-share-obexpush.desktop.in
-help/Makefile
 po/Makefile.in
 src/Makefile
 man/Makefile
