Description: Don't build GFDL-1.1 licensed API documentation files
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = polkitgtkmate src po docs
+SUBDIRS = polkitgtkmate src po
 
 EXTRA_DIST = \
 	autogen.sh \
--- a/configure.ac
+++ b/configure.ac
@@ -207,8 +207,6 @@
 polkitgtkmate/Makefile
 polkitgtkmate/polkit-gtk-mate-1.pc
 src/Makefile
-docs/version.xml
-docs/Makefile
 ])
 
 AC_OUTPUT
@@ -232,6 +230,5 @@
 
         Gtk+ version:               ${GTK_API_VERSION}
         Maintainer mode:            ${USE_MAINTAINER_MODE}
-        Building api docs:          ${enable_gtk_doc}
         GObject Introspection:      ${enable_introspection}
 "
