Description: Don't build non-DFSG (GFDL 1.1 licensed) theme documentation
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
--- a/configure.ac
+++ b/configure.ac
@@ -515,7 +515,6 @@
 AC_CONFIG_FILES([
 Makefile
 doc/Makefile
-doc/creating_themes/Makefile
 doc/man/Makefile
 src/Makefile
 src/wm-tester/Makefile
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = man creating_themes
+SUBDIRS = man
 
 EXTRA_DIST=theme-format.txt marco-theme.dtd dialogs.txt code-overview.txt \
 	how-to-get-focus-right.txt
