--- a/src/org.mate.terminal.gschema.xml.in
+++ b/src/org.mate.terminal.gschema.xml.in
@@ -124,7 +124,7 @@
       <description>If true, allow applications in the terminal to make  text boldface.</description>
     </key>
     <key name="silent-bell" type="b">
-      <default>false</default>
+      <default>true</default>
       <summary>Whether to silence terminal bell</summary>
       <description>If true, don't make a noise when applications send the  escape sequence for the terminal bell.</description>
     </key>
@@ -159,7 +159,7 @@
       <description>Where to put the terminal scrollbar. Possibilities are  "left", "right", and "hidden".</description>
     </key>
     <key name="scrollback-lines" type="i">
-      <default>512</default>
+      <default>10000</default>
       <summary>Number of lines to keep in scrollback</summary>
       <description>Number of scrollback lines to keep around. You can scroll back in the terminal by this number of lines; lines that don't fit in the scrollback are discarded. If scrollback_unlimited is true, this value is ignored.</description>
     </key>
@@ -229,7 +229,7 @@
       <description>An Pango font name. Examples are "Sans 12" or "Monospace Bold 14".</description>
     </key>
     <key name="background-type" enum="org.mate.terminal.background-type">
-      <default>'solid'</default>
+      <default>'transparent'</default>
       <summary>Background type</summary>
       <description>Type of terminal background. May be "solid" for a solid color, "image" for an image, or "transparent" for either real transparency if a compositing window manager is running, or pseudo-transparency otherwise.</description>
     </key>
@@ -244,7 +244,7 @@
       <description>If true, scroll the background image with the foreground  text; if false, keep the image in a fixed position and  scroll the text above it.</description>
     </key>
     <key name="background-darkness" type="d">
-      <default>0.5</default>
+      <default>0.95</default>
       <summary>How much to darken the background image</summary>
       <description>A value between 0.0 and 1.0 indicating how much to darken  the background image. 0.0 means no darkness, 1.0 means fully  dark. In the current implementation, there are only two levels of  darkness possible, so the setting behaves as a boolean, where 0.0 disables the darkening effect.</description>
     </key>
