Description: Don't build non-DFSG (GFDL 1.1 licensed) help files
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
--- a/Makefile.am
+++ b/Makefile.am
@@ -3,7 +3,6 @@
 	libmate-panel-applet		\
 	applets			\
 	doc			\
-	help			\
 	icons			\
 	data			\
 	man			\
--- a/configure.ac
+++ b/configure.ac
@@ -346,9 +346,6 @@
 doc/Makefile
 doc/reference/Makefile
 doc/reference/mate-panel-applet/Makefile
-help/Makefile
-help/mate-clock/Makefile
-help/mate-fish/Makefile
 man/Makefile
 ])
 
